packages/fluent/scss/calendar/_variables.scss

```diff
@@ -282,62 +282,38 @@ $kendo-calendar-end-range-hover-shadow: inset -1px -1px 0 0 if($kendo-enable-col
 
 
 // Calendar sizes
-/// The font size of the small Calendar.
-/// @group calendar
-$kendo-calendar-sm-font-size: $kendo-font-size-sm !default;
-/// The line height of the small Calendar.
-/// @group calendar
-$kendo-calendar-sm-line-height: $kendo-line-height-sm !default;
-/// The size of the small calendar cell.
-/// @group calendar
+$kendo-calendar-sm-font-size: var( --kendo-font-size-sm, inherit ) !default;
```

Comment:
Some of the calendar variables are not documented

Reply:
They actually are https://github.com/telerik/kendo-themes/blob/develop/packages/fluent/scss/calendar/_variables.scss

They have been documented as part of the unification of the Calendar